### PR TITLE
adding opentext to vendor list

### PR DESCRIPTION
**Pull request by kotagirisandeep**
We are introducing Application Observability as a new capability to OpenText Operations Bridge. Release announcement is at https://docs.microfocus.com/doc/OBS/SaaS/ReleaseNotes. Hence adding OpenText as one of the Vendors supporting OpenTelemetry.

### CODE REVIEW

File: data/ecosystem/vendors.yaml [hunk 1]
```diff
@@ -237,6 +237,13 @@
   contact: 'hello@openobserve.ai'
   oss: true
   commercial: true
+- name: OpenText
+  distribution: true
```
Comment (theletterf) [inline data/ecosystem/vendors.yaml:241]: I think this is not a distribution, as you're sending data through any OTel Collector, not your own distribution of it.

File: data/ecosystem/vendors.yaml [hunk 2]
```diff
@@ -237,6 +237,13 @@
...
+  nativeOTLP: true
+  url: 'https://docs.microfocus.com/doc/OBS/SaaS/ConfigureOTELCollector'
+  contact: ''
```
Comment (theletterf) [inline data/ecosystem/vendors.yaml:244]: Could you add a contact email?

**Review by theletterf — CHANGES_REQUESTED** [review]

File: data/ecosystem/vendors.yaml (hunk 1 above)
Comment (kotagirisandeep) [reply data/ecosystem/vendors.yaml:241]: @theletterf at the moment we have our own distribution of OpenTelemetry Collector. Hence set distribution as true.

File: data/ecosystem/vendors.yaml (hunk 2 above)
Comment (kotagirisandeep) [reply data/ecosystem/vendors.yaml]: I added my contact.

**Review by theletterf — APPROVED** [review]
Thanks — all questions clarified. I suggest that you open a separate PR to add the distribution.

**Pull request merged**